package-lock.json

```diff
@@ -3551,7 +3551,7 @@
     "node_modules/html-webpack-plugin/node_modules/json5": {
       "version": "0.5.1",
```

Comment:
It seems that the html-webpack-plugin pulls in a vulnerable version of json5. Maybe we could update this one at least to version 4.x (which should be compatible with v4 of webpack that we use here). In theory, this version of json5 shouldn't be a problem as I think it's only used by webpack in the build process, but still, it would be nice to update.

Reply:
👍